packages/ui/src/components/vuestic-components/va-dropdown/VaDropdown.vue

```diff
@@ -11,17 +11,19 @@
       <slot name="anchor" />
     </div>
     <template v-if="showContent">
-      <div
-        class="va-dropdown__content-wrapper"
-        @mouseover="$props.isContentHoverable && onMouseOver()"
-        @mouseout="onMouseOut()"
-        @click="onDropdownContentClick()"
-        ref="contentWrapper"
-      >
-        <div :style="$props.keepAnchorWidth ? anchorWidthStyles : ''">
-          <slot />
+      <teleport :to="target">
```

Comment:
This demo seems to be broken.

![image](https://user-images.githubusercontent.com/5394573/125784939-26fe97c8-6fcf-4492-a1a3-c36be156dc15.png)

There are generally 2 cases we want to cover:
1) User want dropdown to be global.
2) User wants to keep dropdown inside a wrapper (rare case).
If you think we don't have to consider 2nd case at all - please remove it from demo (mb some other places).